.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
```

Comment:
You might want to skip any commits performed by the system: https://github.com/ExpediaGroup/steerage/blob/master/.github/workflows/release.yml#L7-L11